webapp/src/utils/user_utils.ts

```diff
@@ -18,3 +22,13 @@ export function displayUsernameForUser(user: UserProfile | null, config: Config)
 
     return '';
 }
+
+export function checkMeetingAndServerType(meetingLink:string, useJass:Boolean): boolean {
```

Comment:
Fix lint.
```suggestion
export function checkMeetingAndServerType(meetingLink: string, useJass: boolean): boolean {
```
The function name is not denoting what it is returning. What is the boolean value that is being returned from here?